log/wrapper.go

```diff
@@ -60,3 +61,22 @@ func ZapWrapper(level Level) Wrapper {
 		f(msg)
 	}
 }
+
+// ErrorWithRavenWrapper is a Wrapper implementation that both use Zap logger to
+// log at error level, and send the error to raven.
```

Comment:
Does it make sense to handle the error here twice? I think this deviates a bit from the baseplate.py behavior. I am just wondering if it's the right course of action to surface an error in multiple log systems.

Reply:
Send to raven/sentry will only happen when it's actually configured. For example people usually don't configure it in staging environment, and in those environments the send to sentry part will be a nop. This is why the logging is still useful.